src/pages/assessment/assessmentService.ts

```
@@ -0,0 +1,134 @@
+import { Assessment } from '@migration-planner-ui/api-client/models';
+
+export interface CreateAssessmentRequest {
+  name: string;
+  sourceType: 'inventory' | 'rvtools' | 'agent';
+  inventory?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)